[feat]因"凉宫春日的忧郁:不存在或集数过多"发现的token过期无法正确识别问题的未来新功能建议

资源: 凉宫春日的忧郁 2009 (28集)

不加映射识别命中id: [**485**](https://bgm.tv/subject/485) 不存在或集数过多，跳过
添加映射识别命中id: [**1606**](https://bgm.tv/subject/1606) 不存在或集数过多，跳过

```log
[2026/01/19 09:39:09.568] [INFO] Emby同步任务 emby_1_1768786749 已提交到异步队列
[2026/01/19 09:39:09.569] [INFO] Emby webhook已提交异步任务: emby_1_1768786749
[2026/01/19 09:39:09.568] [DEBUG] 接收到Emby同步请求：{'Title': 'Edge Windows 上 jacksen168 已停止播放 凉宫春日的忧郁 - S1, Ep1 - 朝比奈实玖瑠的冒险 Episode 00', 'Date': '2026-01-19T01:39:09.3601075Z', 'Event': 'playback.stop', 'Severity': 'Info', 'User': {'Name': 'jacksen168', 'Id': '1ac2781c41034c35885e6c29c28b8cd5'}, 'Item': {'Name': '朝比奈实玖瑠的冒险 Episode 00', 'ServerId': '88d3e2edef944171a6a549b81efb8241', 'Id': '6958', 'DateCreated': '2025-04-10T06:32:30.0000000Z', 'DateModified': '2025-04-10T06:32:30.0000000Z', 'Container': 'mkv', 'SortName': '朝比奈实玖瑠的冒险 Episode 00', 'PremiereDate': '2006-04-02T16:00:00.0000000Z', 'ExternalUrls': [{'Name': 'IMDb', 'Url': 'https://www.imdb.com/title/tt0900810'}, {'Name': 'TheTVDB', 'Url': 'https://thetvdb.com/?tab=episode&id=307995'}, {'Name': 'Trakt', 'Url': 'https://trakt.tv/search/imdb/tt0900810'}], 'Path': '/volume2/video/TV shows/凉宫春日的忧郁/Season 01/Suzumiya Haruhi no Yuuutsu 2009 S01E01.mkv', 'OfficialRating': 'TV-14', 'Overview': '“来拍电影吧！”在春日一声令下而拍摄的这部自主制作电影，这一天举办了第一场试映会。为了争夺古泉一树，战斗女服务生（未来人）实玖瑠与邪恶魔法师有希两人，展开了一场又一场的激烈战斗。这场规模浩大的人间戏剧，即将登场？喂，真的假的……\n\n09版第25话。', 'Taglines': [], 'Genres': ['Animation', 'Comedy', 'Drama'], 'CommunityRating': 6.7, 'RunTimeTicks': 14821070000, 'Size': 916188252, 'FileName': 'Suzumiya Haruhi no Yuuutsu 2009 S01E01.mkv', 'Bitrate': 4945328, 'ProductionYear': 2006, 'IndexNumber': 1, 'ParentIndexNumber': 1, 'RemoteTrailers': [], 'ProviderIds': {'Tvdb': '307995', 'Imdb': 'tt0900810'}, 'IsFolder': False, 'ParentId': '6921', 'Type': 'Episode', 'Studios': [], 'GenreItems': [{'Name': 'Animation', 'Id': 1963}, {'Name': 'Comedy', 'Id': 2063}, {'Name': 'Drama', 'Id': 1964}], 'TagItems': [], 'ParentLogoItemId': '6919', 'ParentBackdropItemId': '6919', 'ParentBackdropImageTags': ['bb74f1bf8850461cd65f08e0d11fbc5a'], 'SeriesName': '凉宫春日的忧郁', 'SeriesId': '6919', 'SeasonId': '6921', 'PrimaryImageAspectRatio': 1.7777777777777777, 'SeriesPrimaryImageTag': 'e1d1944ce4b74b920f6132adc7973430', 'SeasonName': '第 1 季', 'ImageTags': {'Primary': '431e0f8c6a982d0759260ff76a83437e'}, 'BackdropImageTags': [], 'ParentLogoImageTag': '7918b1842210c33592bf2eb81f239a1b', 'ParentThumbItemId': '6919', 'ParentThumbImageTag': '1c51234e5fa204de64870e86d98fe70c', 'MediaType': 'Video', 'Width': 1920, 'Height': 1080}, 'Server': {'Name': 'DiskStation', 'Id': '88d3e2edef944171a6a549b81efb8241', 'Version': '4.9.1.90'}, 'Session': {'RemoteEndPoint': '192.168.1.32', 'Client': 'Emby Web', 'DeviceName': 'Edge Windows', 'DeviceId': 'c5283f0e-9349-4272-8b73-e28a790119a8', 'ApplicationVersion': '4.9.1.90', 'Id': 'c53190e723f77c6d5555d52862fb4a71'}, 'PlaybackInfo': {'PlayedToCompletion': True, 'PositionTicks': 14300477710, 'PlaylistIndex': -1, 'PlaylistLength': 0, 'PlaySessionId': '8b3e4c0db9494254954e94c2c8ab2e19', 'MediaSource': {'Chapters': [], 'Protocol': 'File', 'Id': 'mediasource_6958', 'Path': '/volume2/video/TV shows/凉宫春日的忧郁/Season 01/Suzumiya Haruhi no Yuuutsu 2009 S01E01.mkv', 'Type': 'Default', 'Container': 'mkv', 'Size': 916188252, 'Name': 'Suzumiya Haruhi no Yuuutsu 2009 S01E01', 'IsRemote': False, 'HasMixedProtocols': False, 'RunTimeTicks': 14821070000, 'SupportsTranscoding': True, 'SupportsDirectStream': True, 'SupportsDirectPlay': True, 'IsInfiniteStream': False, 'RequiresOpening': False, 'RequiresClosing': False, 'RequiresLooping': False, 'SupportsProbing': True, 'MediaStreams': [{'Codec': 'h264', 'Language': 'jpn', 'TimeBase': '1/1000', 'Title': 'VIDEO_JP', 'VideoRange': 'SDR', 'DisplayTitle': '1080p H264', 'DisplayLanguage': 'Japanese', 'NalLengthSize': '4', 'IsInterlaced': False, 'BitRate': 4945328, 'BitDepth': 8, 'RefFrames': 1, 'IsDefault': True, 'IsForced': False, 'IsHearingImpaired': False, 'Height': 1080, 'Width': 1920, 'AverageFrameRate': 23.976, 'RealFrameRate': 23.976, 'Profile': 'High', 'Type': 'Video', 'AspectRatio': '16:9', 'Index': 0, 'IsExternal': False, 'IsTextSubtitleStream': False, 'SupportsExternalStream': False, 'Protocol': 'File', 'PixelFormat': 'yuv420p', 'Level': 41, 'IsAnamorphic': False, 'ExtendedVideoType': 'None', 'ExtendedVideoSubType': 'None', 'ExtendedVideoSubTypeDescription': 'None', 'AttachmentSize': 0}, {'Codec': 'flac', 'Language': 'jpn', 'TimeBase': '1/1000', 'Title': 'AUDIO_JP', 'DisplayTitle': 'Japanese FLAC stereo (默认)', 'DisplayLanguage': 'Japanese', 'IsInterlaced': False, 'ChannelLayout': 'stereo', 'BitDepth': 16, 'Channels': 2, 'SampleRate': 48000, 'IsDefault': True, 'IsForced': False, 'IsHearingImpaired': False, 'Type': 'Audio', 'Index': 1, 'IsExternal': False, 'IsTextSubtitleStream': False, 'SupportsExternalStream': False, 'Protocol': 'File', 'ExtendedVideoType': 'None', 'ExtendedVideoSubType': 'None', 'ExtendedVideoSubTypeDescription': 'None', 'AttachmentSize': 0}, {'Codec': 'flac', 'Language': 'eng', 'TimeBase': '1/1000', 'Title': 'AUDIO_EN', 'DisplayTitle': 'English FLAC stereo', 'DisplayLanguage': 'English', 'IsInterlaced': False, 'ChannelLayout': 'stereo', 'BitDepth': 16, 'Channels': 2, 'SampleRate': 48000, 'IsDefault': False, 'IsForced': False, 'IsHearingImpaired': False, 'Type': 'Audio', 'Index': 2, 'IsExternal': False, 'IsTextSubtitleStream': False, 'SupportsExternalStream': False, 'Protocol': 'File', 'ExtendedVideoType': 'None', 'ExtendedVideoSubType': 'None', 'ExtendedVideoSubTypeDescription': 'None', 'AttachmentSize': 0}, {'Codec': 'ass', 'Language': 'zh-CN', 'DisplayTitle': 'Chinese Simplified (ASS)', 'DisplayLanguage': 'Chinese Simplified', 'IsInterlaced': False, 'IsDefault': False, 'IsForced': False, 'IsHearingImpaired': False, 'Type': 'Subtitle', 'Index': 3, 'IsExternal': True, 'IsTextSubtitleStream': True, 'SupportsExternalStream': True, 'Path': '/volume2/video/TV shows/凉宫春日的忧郁/Season 01/Suzumiya Haruhi no Yuuutsu 2009 S01E01.chs.ass', 'Protocol': 'File', 'ExtendedVideoType': 'None', 'ExtendedVideoSubType': 'None', 'ExtendedVideoSubTypeDescription': 'None', 'AttachmentSize': 0}], 'Formats': [], 'Bitrate': 4945328, 'RequiredHttpHeaders': {}, 'AddApiKeyToDirectStreamUrl': False, 'ReadAtNativeFramerate': False, 'ItemId': '6958'}}}
[2026/01/19 09:39:09.593] [DEBUG] Emby重新组装JSON报文：media_type='episode' title='凉宫春日的忧郁' ori_title=' ' season=1 episode=1 release_date='2006-04-02' user_name='jacksen168' source='emby'
[2026/01/19 09:39:09.593] [INFO] 接收到同步请求：media_type='episode' title='凉宫春日的忧郁' ori_title=' ' season=1 episode=1 release_date='2006-04-02' user_name='jacksen168' source='emby'
[2026/01/19 09:39:09.596] [DEBUG] 检测到映射配置文件变化，重新加载: /app/config/bangumi_mapping.json
[2026/01/19 09:39:09.596] [DEBUG] 映射配置文件 /app/config/bangumi_mapping.json 中没有配置映射项
[2026/01/19 09:39:09.599] [INFO] 缓存文件已过期（超过 7 天），正在更新...
[2026/01/19 09:39:09.599] [DEBUG] 正在从 https://unpkg.com/bangumi-data@0.3/dist/data.json 下载 bangumi-data...
[2026/01/19 09:39:14.383] [DEBUG] bangumi-data 已缓存到 /app/data/bangumi_data_cache.json
[2026/01/19 09:39:14.385] [INFO] 缓存文件更新成功
[2026/01/19 09:39:14.385] [INFO] 初始化时预加载 bangumi-data 到内存...
[2026/01/19 09:39:15.840] [INFO] 预加载完成，共加载 8378 个项目，耗时 1.46秒
[2026/01/19 09:39:15.841] [DEBUG] 正在查找番剧 ID: title='凉宫春日的忧郁', ori_title=' ', release_date='2006-04-02', season=1
[2026/01/19 09:39:15.841] [DEBUG] 开始尝试完全匹配...
[2026/01/19 09:39:15.841] [DEBUG] 使用内存缓存数据 (命中次数: 1)
[2026/01/19 09:39:17.145] [DEBUG] 处理了 8378 个项目，找到 2 个完全匹配，2 个部分匹配
[2026/01/19 09:39:17.145] [DEBUG] 找到最佳日期匹配的番剧: 涼宮ハルヒの憂鬱, bangumi_id: 485
[2026/01/19 09:39:17.145] [INFO] 通过 bangumi-data 匹配到番剧 ID: 485, 匹配标题: 凉宫春日的忧郁
[2026/01/19 09:39:17.146] [INFO] 📤 发送 bangumi_id_found 通知到: https://anime.jacksen168.top/api/data/setWatchedStatus.php
[2026/01/19 09:39:17.779] [INFO] ✅ Webhook通知发送成功，响应状态码: 200
[2026/01/19 09:39:17.781] [WARNING] SSL证书验证已禁用，这会降低安全性。建议仅在代理环境下出现SSL错误时使用。
[2026/01/19 09:39:17.782] [DEBUG] BangumiApi 初始化 - 代理参数: http://172.17.0.1:7890, SSL验证: False
[2026/01/19 09:39:17.782] [DEBUG] 直接尝试从指定季度ID匹配集数: 485, 目标季度: 1, 目标集数: 1
[2026/01/19 09:39:17.782] [DEBUG] BangumiApi GET请求: https://api.bgm.tv/v0/episodes, 代理: {'http': 'http://172.17.0.1:7890', 'https': 'http://172.17.0.1:7890'}
[2026/01/19 09:39:19.062] [DEBUG] []
[2026/01/19 09:39:19.062] [DEBUG] 未获取到剧集信息: 485
[2026/01/19 09:39:19.062] [ERROR] bgm: subject_id='485' item.season=1 item.episode=1, 不存在或集数过多，跳过
[2026/01/19 09:41:43.441] [DEBUG] 接收到Emby同步请求：{'Title': 'DiskStation 上 jacksen168 已标记 凉宫春日的忧郁 - S1, Ep1 - 朝比奈实玖瑠的冒险 Episode 00 为已播放', 'Date': '2026-01-19T01:41:43.3865919Z', 'Event': 'item.markplayed', 'Severity': 'Info', 'User': {'Name': 'jacksen168', 'Id': '1ac2781c41034c35885e6c29c28b8cd5'}, 'Item': {'Name': '朝比奈实玖瑠的冒险 Episode 00', 'ServerId': '88d3e2edef944171a6a549b81efb8241', 'Id': '6958', 'DateCreated': '2025-04-10T06:32:30.0000000Z', 'DateModified': '2025-04-10T06:32:30.0000000Z', 'Container': 'mkv', 'SortName': '朝比奈实玖瑠的冒险 Episode 00', 'PremiereDate': '2006-04-02T16:00:00.0000000Z', 'ExternalUrls': [{'Name': 'IMDb', 'Url': 'https://www.imdb.com/title/tt0900810'}, {'Name': 'TheTVDB', 'Url': 'https://thetvdb.com/?tab=episode&id=307995'}, {'Name': 'Trakt', 'Url': 'https://trakt.tv/search/imdb/tt0900810'}], 'Path': '/volume2/video/TV shows/凉宫春日的忧郁/Season 01/Suzumiya Haruhi no Yuuutsu 2009 S01E01.mkv', 'OfficialRating': 'TV-14', 'Overview': '“来拍电影吧！”在春日一声令下而拍摄的这部自主制作电影，这一天举办了第一场试映会。为了争夺古泉一树，战斗女服务生（未来人）实玖瑠与邪恶魔法师有希两人，展开了一场又一场的激烈战斗。这场规模浩大的人间戏剧，即将登场？喂，真的假的……\n\n09版第25话。', 'Taglines': [], 'Genres': ['Animation', 'Comedy', 'Drama'], 'CommunityRating': 6.7, 'RunTimeTicks': 14821070000, 'Size': 916188252, 'FileName': 'Suzumiya Haruhi no Yuuutsu 2009 S01E01.mkv', 'Bitrate': 4945328, 'ProductionYear': 2006, 'IndexNumber': 1, 'ParentIndexNumber': 1, 'RemoteTrailers': [], 'ProviderIds': {'Tvdb': '307995', 'Imdb': 'tt0900810'}, 'IsFolder': False, 'ParentId': '6921', 'Type': 'Episode', 'Studios': [], 'GenreItems': [{'Name': 'Animation', 'Id': 1963}, {'Name': 'Comedy', 'Id': 2063}, {'Name': 'Drama', 'Id': 1964}], 'TagItems': [], 'ParentLogoItemId': '6919', 'ParentBackdropItemId': '6919', 'ParentBackdropImageTags': ['bb74f1bf8850461cd65f08e0d11fbc5a'], 'UserData': {'PlaybackPositionTicks': 0, 'PlayCount': 1, 'IsFavorite': False, 'LastPlayedDate': '2026-01-19T01:41:43.0000000Z', 'Played': True}, 'SeriesName': '凉宫春日的忧郁', 'SeriesId': '6919', 'SeasonId': '6921', 'PrimaryImageAspectRatio': 1.7777777777777777, 'SeriesPrimaryImageTag': 'e1d1944ce4b74b920f6132adc7973430', 'SeasonName': '第 1 季', 'ImageTags': {'Primary': '431e0f8c6a982d0759260ff76a83437e'}, 'BackdropImageTags': [], 'ParentLogoImageTag': '7918b1842210c33592bf2eb81f239a1b', 'ParentThumbItemId': '6919', 'ParentThumbImageTag': '1c51234e5fa204de64870e86d98fe70c', 'MediaType': 'Video', 'Width': 1920, 'Height': 1080}, 'Server': {'Name': 'DiskStation', 'Id': '88d3e2edef944171a6a549b81efb8241', 'Version': '4.9.1.90'}}
[2026/01/19 09:41:43.441] [DEBUG] Emby重新组装JSON报文：media_type='episode' title='凉宫春日的忧郁' ori_title=' ' season=1 episode=1 release_date='2006-04-02' user_name='jacksen168' source='emby'
[2026/01/19 09:41:43.441] [INFO] 接收到同步请求：media_type='episode' title='凉宫春日的忧郁' ori_title=' ' season=1 episode=1 release_date='2006-04-02' user_name='jacksen168' source='emby'
[2026/01/19 09:41:43.441] [INFO] Emby同步任务 emby_2_1768786903 已提交到异步队列
[2026/01/19 09:41:43.441] [INFO] Emby webhook已提交异步任务: emby_2_1768786903
[2026/01/19 09:41:43.446] [DEBUG] 检测到映射配置文件变化，重新加载: /app/config/bangumi_mapping.json
[2026/01/19 09:41:43.447] [DEBUG] 映射配置文件 /app/config/bangumi_mapping.json 中没有配置映射项
[2026/01/19 09:41:43.447] [DEBUG] 正在查找番剧 ID: title='凉宫春日的忧郁', ori_title=' ', release_date='2006-04-02', season=1
[2026/01/19 09:41:43.447] [DEBUG] 开始尝试完全匹配...
[2026/01/19 09:41:43.447] [DEBUG] 使用内存缓存数据 (命中次数: 2)
[2026/01/19 09:41:44.768] [DEBUG] 处理了 8378 个项目，找到 2 个完全匹配，2 个部分匹配
[2026/01/19 09:41:44.768] [DEBUG] 找到最佳日期匹配的番剧: 涼宮ハルヒの憂鬱, bangumi_id: 485
[2026/01/19 09:41:44.768] [INFO] 通过 bangumi-data 匹配到番剧 ID: 485, 匹配标题: 凉宫春日的忧郁
[2026/01/19 09:41:44.769] [INFO] 📤 发送 bangumi_id_found 通知到: https://anime.jacksen168.top/api/data/setWatchedStatus.php
[2026/01/19 09:41:45.393] [INFO] ✅ Webhook通知发送成功，响应状态码: 200
[2026/01/19 09:41:45.395] [WARNING] SSL证书验证已禁用，这会降低安全性。建议仅在代理环境下出现SSL错误时使用。
[2026/01/19 09:41:45.395] [DEBUG] BangumiApi 初始化 - 代理参数: http://172.17.0.1:7890, SSL验证: False
[2026/01/19 09:41:45.395] [DEBUG] 直接尝试从指定季度ID匹配集数: 485, 目标季度: 1, 目标集数: 1
[2026/01/19 09:41:45.395] [DEBUG] BangumiApi GET请求: https://api.bgm.tv/v0/episodes, 代理: {'http': 'http://172.17.0.1:7890', 'https': 'http://172.17.0.1:7890'}
[2026/01/19 09:41:47.227] [DEBUG] []
[2026/01/19 09:41:47.227] [DEBUG] 未获取到剧集信息: 485
[2026/01/19 09:41:47.227] [ERROR] bgm: subject_id='485' item.season=1 item.episode=1, 不存在或集数过多，跳过
[2026/01/19 09:43:16.275] [DEBUG] 已隐藏加密密码，返回空字符串给前端
[2026/01/19 09:43:19.055] [DEBUG] 检测到映射配置文件变化，重新加载: /app/config/bangumi_mapping.json
[2026/01/19 09:43:19.056] [DEBUG] 映射配置文件 /app/config/bangumi_mapping.json 中没有配置映射项
[2026/01/19 09:43:21.167] [DEBUG] 检测到映射配置文件变化，重新加载: /app/config/bangumi_mapping.json
[2026/01/19 09:43:21.167] [DEBUG] 映射配置文件 /app/config/bangumi_mapping.json 中没有配置映射项
[2026/01/19 09:46:18.310] [INFO] 强制重新加载自定义映射配置
[2026/01/19 09:46:18.310] [DEBUG] 检测到映射配置文件变化，重新加载: /app/config/bangumi_mapping.json
[2026/01/19 09:46:18.310] [DEBUG] 从 /app/config/bangumi_mapping.json 重新加载了 1 个自定义映射
[2026/01/19 09:46:18.311] [INFO] 自定义映射已更新，共 1 个映射
[2026/01/19 09:46:18.337] [DEBUG] 使用缓存的映射配置，共 1 个映射
[2026/01/19 09:46:28.552] [DEBUG] 接收到Emby同步请求：{'Title': 'DiskStation 上 jacksen168 已标记 凉宫春日的忧郁 - S1, Ep1 - 朝比奈实玖瑠的冒险 Episode 00 为已播放', 'Date': '2026-01-19T01:46:28.4977724Z', 'Event': 'item.markplayed', 'Severity': 'Info', 'User': {'Name': 'jacksen168', 'Id': '1ac2781c41034c35885e6c29c28b8cd5'}, 'Item': {'Name': '朝比奈实玖瑠的冒险 Episode 00', 'ServerId': '88d3e2edef944171a6a549b81efb8241', 'Id': '6958', 'DateCreated': '2025-04-10T06:32:30.0000000Z', 'DateModified': '2025-04-10T06:32:30.0000000Z', 'Container': 'mkv', 'SortName': '朝比奈实玖瑠的冒险 Episode 00', 'PremiereDate': '2006-04-02T16:00:00.0000000Z', 'ExternalUrls': [{'Name': 'IMDb', 'Url': 'https://www.imdb.com/title/tt0900810'}, {'Name': 'TheTVDB', 'Url': 'https://thetvdb.com/?tab=episode&id=307995'}, {'Name': 'Trakt', 'Url': 'https://trakt.tv/search/imdb/tt0900810'}], 'Path': '/volume2/video/TV shows/凉宫春日的忧郁/Season 01/Suzumiya Haruhi no Yuuutsu 2009 S01E01.mkv', 'OfficialRating': 'TV-14', 'Overview': '“来拍电影吧！”在春日一声令下而拍摄的这部自主制作电影，这一天举办了第一场试映会。为了争夺古泉一树，战斗女服务生（未来人）实玖瑠与邪恶魔法师有希两人，展开了一场又一场的激烈战斗。这场规模浩大的人间戏剧，即将登场？喂，真的假的……\n\n09版第25话。', 'Taglines': [], 'Genres': ['Animation', 'Comedy', 'Drama'], 'CommunityRating': 6.7, 'RunTimeTicks': 14821070000, 'Size': 916188252, 'FileName': 'Suzumiya Haruhi no Yuuutsu 2009 S01E01.mkv', 'Bitrate': 4945328, 'ProductionYear': 2006, 'IndexNumber': 1, 'ParentIndexNumber': 1, 'RemoteTrailers': [], 'ProviderIds': {'Tvdb': '307995', 'Imdb': 'tt0900810'}, 'IsFolder': False, 'ParentId': '6921', 'Type': 'Episode', 'Studios': [], 'GenreItems': [{'Name': 'Animation', 'Id': 1963}, {'Name': 'Comedy', 'Id': 2063}, {'Name': 'Drama', 'Id': 1964}], 'TagItems': [], 'ParentLogoItemId': '6919', 'ParentBackdropItemId': '6919', 'ParentBackdropImageTags': ['bb74f1bf8850461cd65f08e0d11fbc5a'], 'UserData': {'PlaybackPositionTicks': 0, 'PlayCount': 1, 'IsFavorite': False, 'LastPlayedDate': '2026-01-19T01:46:28.0000000Z', 'Played': True}, 'SeriesName': '凉宫春日的忧郁', 'SeriesId': '6919', 'SeasonId': '6921', 'PrimaryImageAspectRatio': 1.7777777777777777, 'SeriesPrimaryImageTag': 'e1d1944ce4b74b920f6132adc7973430', 'SeasonName': '第 1 季', 'ImageTags': {'Primary': '431e0f8c6a982d0759260ff76a83437e'}, 'BackdropImageTags': [], 'ParentLogoImageTag': '7918b1842210c33592bf2eb81f239a1b', 'ParentThumbItemId': '6919', 'ParentThumbImageTag': '1c51234e5fa204de64870e86d98fe70c', 'MediaType': 'Video', 'Width': 1920, 'Height': 1080}, 'Server': {'Name': 'DiskStation', 'Id': '88d3e2edef944171a6a549b81efb8241', 'Version': '4.9.1.90'}}
[2026/01/19 09:46:28.552] [DEBUG] Emby重新组装JSON报文：media_type='episode' title='凉宫春日的忧郁' ori_title=' ' season=1 episode=1 release_date='2006-04-02' user_name='jacksen168' source='emby'
[2026/01/19 09:46:28.552] [INFO] 接收到同步请求：media_type='episode' title='凉宫春日的忧郁' ori_title=' ' season=1 episode=1 release_date='2006-04-02' user_name='jacksen168' source='emby'
[2026/01/19 09:46:28.553] [DEBUG] 使用缓存的映射配置，共 1 个映射
[2026/01/19 09:46:28.553] [DEBUG] 匹配到自定义映射：凉宫春日的忧郁=1606
[2026/01/19 09:46:28.553] [INFO] 📤 发送 bangumi_id_found 通知到: https://anime.jacksen168.top/api/data/setWatchedStatus.php
[2026/01/19 09:46:28.552] [INFO] Emby同步任务 emby_3_1768787188 已提交到异步队列
[2026/01/19 09:46:28.555] [INFO] Emby webhook已提交异步任务: emby_3_1768787188
[2026/01/19 09:46:29.220] [INFO] ✅ Webhook通知发送成功，响应状态码: 200
[2026/01/19 09:46:29.222] [WARNING] SSL证书验证已禁用，这会降低安全性。建议仅在代理环境下出现SSL错误时使用。
[2026/01/19 09:46:29.223] [DEBUG] BangumiApi 初始化 - 代理参数: http://172.17.0.1:7890, SSL验证: False
[2026/01/19 09:46:29.223] [DEBUG] BangumiApi GET请求: https://api.bgm.tv/v0/episodes, 代理: {'http': 'http://172.17.0.1:7890', 'https': 'http://172.17.0.1:7890'}
[2026/01/19 09:46:31.652] [DEBUG] 未获取到剧集信息: 1606
[2026/01/19 09:46:31.653] [ERROR] bgm: subject_id='1606' item.season=1 item.episode=1, 不存在或集数过多，跳过
[2026/01/19 09:48:14.965] [DEBUG] 已隐藏加密密码，返回空字符串给前端
[2026/01/19 09:48:27.288] [INFO] 更新webhook配置成功: ID=2
[2026/01/19 09:48:39.821] [INFO] Emby同步任务 emby_4_1768787319 已提交到异步队列
[2026/01/19 09:48:39.821] [INFO] Emby webhook已提交异步任务: emby_4_1768787319
[2026/01/19 09:48:39.824] [DEBUG] 接收到Emby同步请求：{'Title': 'DiskStation 上 jacksen168 已标记 凉宫春日的忧郁 - S1, Ep2 - 凉宫春日的忧郁Ⅰ 为已播放', 'Date': '2026-01-19T01:48:39.7680276Z', 'Event': 'item.markplayed', 'Severity': 'Info', 'User': {'Name': 'jacksen168', 'Id': '1ac2781c41034c35885e6c29c28b8cd5'}, 'Item': {'Name': '凉宫春日的忧郁Ⅰ', 'ServerId': '88d3e2edef944171a6a549b81efb8241', 'Id': '6959', 'DateCreated': '2025-04-10T06:41:19.0000000Z', 'DateModified': '2025-04-10T06:41:19.0000000Z', 'Container': 'mkv', 'SortName': '凉宫春日的忧郁Ⅰ', 'PremiereDate': '2006-04-09T16:00:00.0000000Z', 'ExternalUrls': [{'Name': 'IMDb', 'Url': 'https://www.imdb.com/title/tt0900818'}, {'Name': 'TheTVDB', 'Url': 'https://thetvdb.com/?tab=episode&id=307996'}, {'Name': 'Trakt', 'Url': 'https://trakt.tv/search/imdb/tt0900818'}], 'Path': '/volume2/video/TV shows/凉宫春日的忧郁/Season 01/Suzumiya Haruhi no Yuuutsu 2009 S01E02.mkv', 'OfficialRating': 'TV-14', 'Overview': '阿虚曾几何时，都希望过世上真的有外星人未来人、超能力者、幽灵妖怪等超自然生物，但明白这是天方夜谈之后，他早已习惯了平凡的生活，直到一天他遇上凉宫春日……凉宫春日虽然是个超级美少女，但思想行为怪异，而且超自我中心，只希望与外星人或同等级数的人做朋友，阿虚可以说是唯一例的人。一天，她被阿虚一言惊醒，决定要创立一个新学会，来满足自己的欲望，SOS团因此而成立!\n\n09版第01话。', 'Taglines': [], 'Genres': ['Animation', 'Comedy', 'Fantasy'], 'CommunityRating': 8.3, 'RunTimeTicks': 14049890000, 'Size': 823605125, 'FileName': 'Suzumiya Haruhi no Yuuutsu 2009 S01E02.mkv', 'Bitrate': 4689603, 'ProductionYear': 2006, 'IndexNumber': 2, 'ParentIndexNumber': 1, 'RemoteTrailers': [], 'ProviderIds': {'Tvdb': '307996', 'Imdb': 'tt0900818'}, 'IsFolder': False, 'ParentId': '6921', 'Type': 'Episode', 'Studios': [], 'GenreItems': [{'Name': 'Animation', 'Id': 1963}, {'Name': 'Comedy', 'Id': 2063}, {'Name': 'Fantasy', 'Id': 1965}], 'TagItems': [], 'ParentLogoItemId': '6919', 'ParentBackdropItemId': '6919', 'ParentBackdropImageTags': ['bb74f1bf8850461cd65f08e0d11fbc5a'], 'UserData': {'PlaybackPositionTicks': 0, 'PlayCount': 1, 'IsFavorite': False, 'LastPlayedDate': '2026-01-19T01:48:39.0000000Z', 'Played': True}, 'SeriesName': '凉宫春日的忧郁', 'SeriesId': '6919', 'SeasonId': '6921', 'PrimaryImageAspectRatio': 1.7777777777777777, 'SeriesPrimaryImageTag': 'e1d1944ce4b74b920f6132adc7973430', 'SeasonName': '第 1 季', 'ImageTags': {'Primary': '3988eff8f0c66f1227af8049ff4871a0'}, 'BackdropImageTags': [], 'ParentLogoImageTag': '7918b1842210c33592bf2eb81f239a1b', 'ParentThumbItemId': '6919', 'ParentThumbImageTag': '1c51234e5fa204de64870e86d98fe70c', 'MediaType': 'Video', 'Width': 1920, 'Height': 1080}, 'Server': {'Name': 'DiskStation', 'Id': '88d3e2edef944171a6a549b81efb8241', 'Version': '4.9.1.90'}}
[2026/01/19 09:48:39.825] [DEBUG] Emby重新组装JSON报文：media_type='episode' title='凉宫春日的忧郁' ori_title=' ' season=1 episode=2 release_date='2006-04-09' user_name='jacksen168' source='emby'
[2026/01/19 09:48:39.825] [INFO] 接收到同步请求：media_type='episode' title='凉宫春日的忧郁' ori_title=' ' season=1 episode=2 release_date='2006-04-09' user_name='jacksen168' source='emby'
[2026/01/19 09:48:39.825] [DEBUG] 使用缓存的映射配置，共 1 个映射
[2026/01/19 09:48:39.825] [DEBUG] 匹配到自定义映射：凉宫春日的忧郁=1606
[2026/01/19 09:48:39.826] [INFO] 📤 发送 bangumi_id_found 通知到: https://anime.jacksen168.top/api/data/setWatchedStatus.php
[2026/01/19 09:48:40.497] [INFO] ✅ Webhook通知发送成功，响应状态码: 200
[2026/01/19 09:48:40.499] [WARNING] SSL证书验证已禁用，这会降低安全性。建议仅在代理环境下出现SSL错误时使用。
[2026/01/19 09:48:40.499] [DEBUG] BangumiApi 初始化 - 代理参数: http://172.17.0.1:7890, SSL验证: False
[2026/01/19 09:48:40.499] [DEBUG] BangumiApi GET请求: https://api.bgm.tv/v0/episodes, 代理: {'http': 'http://172.17.0.1:7890', 'https': 'http://172.17.0.1:7890'}
[2026/01/19 09:48:41.321] [DEBUG] 未获取到剧集信息: 1606
[2026/01/19 09:48:41.321] [ERROR] bgm: subject_id='1606' item.season=1 item.episode=2, 不存在或集数过多，跳过
```

--- a/app/services/sync_service.py
+++ b/app/services/sync_service.py
@@ -199,12 +199,19 @@ class SyncService:
                 return SyncResponse(status="error", message="bangumi配置错误")
 
             # 查询bangumi番剧指定季度指定集数信息
-            bgm_se_id, bgm_ep_id = bgm.get_target_season_episode_id(
-                subject_id=subject_id,
-                target_season=item.season,
-                target_ep=item.episode,
-                is_season_subject_id=is_season_matched_id,
-            )
+            try:
+                bgm_se_id, bgm_ep_id = bgm.get_target_season_episode_id(
+                    subject_id=subject_id,
+                    target_season=item.season,
+                    target_ep=item.episode,
+                    is_season_subject_id=is_season_matched_id,
+                )
+            except ValueError as ve:
+                # 捕获认证错误（通知已在 BangumiApi 中发送）
+                if "认证失败" in str(ve) or "access_token" in str(ve):
+                    return SyncResponse(status="error", message=str(ve))
+                else:
+                    raise ve
 
             if not bgm_ep_id:
                 logger.error(
@@ -218,7 +225,15 @@ class SyncService:
             )
 
             # 标记为看过
-            mark_status = self._retry_mark_episode(bgm, bgm_se_id, bgm_ep_id)
+            try:
+                mark_status = self._retry_mark_episode(bgm, bgm_se_id, bgm_ep_id)
+            except ValueError as ve:
+                # 捕获认证错误（通知已在 BangumiApi 中发送）
+                if "认证失败" in str(ve) or "access_token" in str(ve):
+                    return SyncResponse(status="error", message=str(ve))
+                else:
+                    raise ve
+
             result_message = ""
 
             if mark_status == 0:

--- a/app/utils/bangumi_api.py
+++ b/app/utils/bangumi_api.py
@@ -289,31 +289,63 @@ class BangumiApi:
 
                     raise e
 
+    def _check_auth_error(self, res):
+        """统一检查认证错误"""
+        if res.status_code == 401:
+            error_msg = "Bangumi API 认证失败: access_token可能已过期（有效期1年）或无效，请更新token"
+            logger.error(error_msg)
+
+            # 发送API认证失败通知（webhook和邮件）
+            try:
+                from .notifier import get_notifier
+
+                notifier = get_notifier()
+                notifier.send_notification_by_type(
+                    "api_auth_error",
+                    {
+                        "timestamp": datetime.datetime.now().strftime(
+                            "%Y-%m-%d %H:%M:%S"
+                        ),
+                        "username": self.username,
+                        "status_code": res.status_code,
+                        "error_message": error_msg,
+                    },
+                )
+            except Exception as notify_error:
+                logger.error(f"发送API认证失败通知失败: {notify_error}")
+
+            raise ValueError(error_msg)
+        return res
+
     def get(self, path, params=None):
         logger.debug(
             f"BangumiApi GET请求: {self.host}/{path}, 代理: {self.req.proxies if self.req.proxies else '无'}"
         )
-        return self._request_with_retry(
+        res = self._request_with_retry(
             "GET", self.req, f"{self.host}/{path}", params=params
         )
+        return self._check_auth_error(res)
 
     def post(self, path, _json, params=None):
         logger.debug(
             f"BangumiApi POST请求: {self.host}/{path}, 代理: {self.req.proxies if self.req.proxies else '无'}"
         )
-        return self._request_with_retry(
+        res = self._request_with_retry(
             "POST", self.req, f"{self.host}/{path}", json=_json, params=params
         )
+        return self._check_auth_error(res)
 
     def put(self, path, _json, params=None):
-        return self._request_with_retry(
+        res = self._request_with_retry(
             "PUT", self.req, f"{self.host}/{path}", json=_json, params=params
         )
+        return self._check_auth_error(res)
 
     def patch(self, path, _json, params=None):
-        return self._request_with_retry(
+        res = self._request_with_retry(
             "PATCH", self.req, f"{self.host}/{path}", json=_json, params=params
         )
+        return self._check_auth_error(res)
 
     def get_me(self):
         res = self.get("me")

